docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc

```diff
@@ -44,16 +44,16 @@ Full List of Resources:
     * haproxy-bundle-podman-2   (ocf:heartbeat:podman):  Started controller-1
 ----
 
-. Use the `ip` command to identify the ranges of the storage networks.
+. Identify the ranges of the storage networks:
```

Comment:
You could add something like "The following is an example and the values might differ in your environment"

Maybe we have a boilerplate sentence for this?